crates/kobold/js/util.js

```diff
@@ -22,22 +17,16 @@ export function __kobold_fragment_unmount(f)
 {
 	let [b, e] = fragmentDecorators.get(f);
 	while (b.nextSibling !== e) f.appendChild(b.nextSibling);
-	f.appendChild(e);
-	f.insertBefore(b, f.firstChild);
+	f.appendChild(e).insertBefore(b, f.firstChild);
 }
 export function __kobold_fragment_replace(f,n)
 {
 	let [b, e] = fragmentDecorators.get(f);
 	while (b.nextSibling !== e) f.appendChild(b.nextSibling);
 	b.replaceWith(n);
-	f.appendChild(e);
-	f.insertBefore(b, f.firstChild);
-}
-export function __kobold_fragment_drop(f)
-{
-	delete f.$begin;
-	delete f.$end;
+	f.appendChild(e).insertBefore(b, f.firstChild);
 }
+export function __kobold_fragment_drop(f) { delete f.$begin & delete f.$end; }
```

Comment:
This is fine but I also kind of hate it :sweat_smile:

Reply:
yes i also prefer multi-line as it's more readable